pkg/hub/controllers/placemanifestworkcontroller/placemanifestwork_deploy_reconcile.go

```diff
@@ -18,5 +27,92 @@ type deployReconciler struct {
 
 func (d *deployReconciler) reconcile(ctx context.Context, pw *workapiv1alpha1.PlaceManifestWork) (*workapiv1alpha1.PlaceManifestWork, reconcileState, error) {
 	// TODO the main manifestwork create/update/delete logic, we may also need to manage the status condition for whether manifestwork is created here
+	klog.Info("reconciling placeManifestWork ", pw.Name)
+
+	placement, err := d.placementLister.Placements(pw.Namespace).Get(pw.Spec.PlacementRef.Name)
+	if err != nil {
+		//klog.Info("my reconciling placement err ", err)
+		apimeta.SetStatusCondition(&pw.Status.Conditions, GetPlacementDecisionVerified(workapiv1alpha1.PlacementDecisionNotFound, ""))
+
+		return pw, reconcileContinue, fmt.Errorf("Failed get placement %w", err)
+	}
+
+	//klog.Info("my reconciling placement ", placement)
+	// TODO: Could be better to change the pw status to include list of selected clusters otherwise we need to list all the manifestworks belong to pw to figure which cluster Added or deleted
```

Comment:
I think get manifestwork by lister is good. We cannot record clusters in status, since it could be many clusters and change frequently. Get by lister won't produce api call so it is appropriate here.